lib/rules/no-constant-condition.js

```diff
@@ -46,7 +45,7 @@ module.exports = {
 
     create(context) {
         const options = context.options[0] || {},
-            checkLoops = options.checkLoops !== false,
+            checkLoops = options.checkLoops ?? "allExceptWhileTrue",
```

Comment:
I like to normalize the bool-style to the new options, so the following code can be unchanged. something like:

```js
if(options.checkLoops === true) checkLoops = "all";
if(options.checkLoops === false) checkLoops = "none";
```

Reply:
Done!